packages/library/src/commonMain/kotlin/io/realm/RealmConfiguration.kt

```diff
@@ -121,6 +122,11 @@ public class RealmConfiguration private constructor(
      */
     public val deleteRealmIfMigrationNeeded: Boolean
 
+    /**
+     * 64 byte key used to encrypt and decrypt the Realm file.
+     */
+    public val encryptionKey get() = RealmInterop.realm_config_get_encryption_key(nativeConfig)
```

Comment:
We should probably make the return type explicit here for better discoverability.